### PR TITLE
added a comment on sorting stories written in storiesOf

### PR DESCRIPTION
**Pull request by imri29**
Added a comment under the sorting stories example. I had a hard time understanding why that didn't work for me, and following a conversation with @shilman on Discord I've added this comment.

Phrasing was decided by me, naturally I imagine you have better ideas of what it should be.

### CODE REVIEW

File: content/design-systems-for-developers/react/en/document.md [hunk 1]
```diff
@@ -364,6 +364,9 @@ module.exports = {
 };
 ```
 
+Please note, if your stories are written in the `storiesOf` format, you'll need to sort them with the `sortStories` function.
+More details can be found [here](https://storybook.js.org/docs/configurations/options-parameter/).
+
```
Comment (jonniebigodes) [inline content/design-systems-for-developers/react/en/document.md]: @imri29 before going into this if you allow me some reference here. I'm assuming that you're still using the `storiesOf` format to some extent correct. Or you were using it?
If that's the case, i think we have room for improvement here. We can probably fine tune it into something like:



```html
<div class="aside">
 <p>If you're still using <code>storiesOf</code> format, you'll need to make some adjustments to your <code>./storybook/preview.js</code> and add the  <code>sortStories</code> function in order to display them correctly.</p>
 <p> You can read more about it <a href="https://storybook.js.org/docs/configurations/options-parameter/">here</p>
</div>
```
With this tweak we can keep the flow of the documentation intact. But making a note to your case and probably other people that might come into this issue.

Also @domyen and @tmeasday i would like your input aswell as you were the ones that wrote the tutorial.

If you're all ok with this and @imri29 once you manage to address the comment i wouldn't mind merging this.

Feel free to provide feedback

Comment (domyen) [reply content/design-systems-for-developers/react/en/document.md]: Thanks @imri29. This is a good tweak @jonniebigodes! Happy to merge it.

Comment (imri29) [reply content/design-systems-for-developers/react/en/document.md]: @jonniebigodes thanks for the improvement. I did think about putting it in an `aside` at first but wasn't sure if that was the right approach. 
I've pushed the revision now. Thanks!

Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/document.md]: @imri29 no need to thank, glad that i was able to help you fine tune this. I've just checked the latest commit and it looks good. Me personally and i believe the rest of the team is more than grateful for this small but rather important contribution. I'll defer to @domyen to merge this then.

File: content/design-systems-for-developers/react/en/document.md [hunk 2]
```diff
@@ -364,6 +364,11 @@ module.exports = {
 };
 ```
 
+<div class="aside">
+ <p>If you're still using <code>storiesOf</code> format, you'll need to make some adjustments to your <code>./storybook/preview.js</code> and add the  <code>sortStories</code> function in order to display them correctly.</p>
+ <p> You can read more about it <a href="https://storybook.js.org/docs/configurations/options-parameter/">here.</a></p>
```
Comment (domyen) [inline content/design-systems-for-developers/react/en/document.md]:
```suggestion
 <p> You can read more about it <a href="https://storybook.js.org/docs/configurations/options-parameter/#sorting-stories">here.</a></p>
```

Comment (domyen) [reply content/design-systems-for-developers/react/en/document.md]: One last thing @imri29, let's directly link to the section in the docs for this!

**Pull request merged**